org-code-javabuilder/playground/src/main/java/org/code/playground/Item.java

```diff
@@ -0,0 +1,22 @@
+package org.code.playground;
+
+import java.io.FileNotFoundException;
+
+public class Item {
+  
+   /**
+   * Creates an item that can be displayed in the Playground. An item simply consists of an image,
+   * referenced by the name of the image file in the asset manager.
+   *
+   * @param filename the string name of the color (case insensitive).
+   * @throws FileNotFoundException if the file specified is not the in the asset manager
+   */
+  public Item(String filename) throws FileNotFoundException;
```

Comment:
should the x, y width and height values live on `Item` instead of being set when the Item is added to the board? How would a click event change the x, y, width or height? Also should we have a setFilename option?

Reply:
Just chatted with Jamila on these and I had incorrectly thought that putting them on the API call was preferable, but that's not the case. I'll be changing these soon!

Reply:
Ah interesting, my impression was that the only way to change an item's position/size was to remove it and add it with a different position/size, but with this change it seems like that's not the case?

Reply:
Yes, now students should be able to move and resize things in this API.